tests/index.js

```diff
@@ -121,7 +121,34 @@ describe('git-repo-info', function() {
         author: null,
         authorDate: null,
         commitMessage: null,
-        root: repoRoot
+        root: repoRoot,
+        lastTag: null,
+        noOfAdditionalCommits: Infinity
+      };
+
+      assert.deepEqual(result, expected);
+    });
+
+    it('returns an object with repo info including the parent tag', function() {
+      var repoRoot = path.join(testFixturesPath, 'tag-on-parent');
+      var result = repoInfo(path.join(repoRoot, gitDir));
+
+      var expected = {
+        branch: 'master',
+        sha: 'fb26504da0ed5cd9ed366f7428c06a8433fd76e6',
+        abbreviatedSha: 'fb26504da0',
+        tag: null,
+        committer: 'Lukas Kohler <lukas.kohler@ontheblueplanet.com>',
+        committerDate: '2017-10-14T02:02:43.000Z',
+        author: 'Lukas Kohler <lukas.kohler@ontheblueplanet.com>',
+        authorDate: '2017-10-14T02:02:43.000Z',
+        commitMessage: 'second commit without tag',
+        root: repoRoot,
+        parents: [
+          'e66f7ec2da3b5d06f0fe845c4fbc87247efacf62'
```

Comment:
maybe add a test to handle the merge commit case (ie multiple parents) since it looks like you already added code to make that work.  What do you think?

Reply:
I've added a test for the simple case where just one path has a tag. However its not clear what should happen if both paths have a tag (on the same distance). However I dont think we have to figure out all theese edge cases. For now I think its OK to not guarantee a specific behavior in that case.

Reply:
@luxferresum yes I agree it's really not clear what the right answer is there.